packages/expo-ui/components/List/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { requireNativeView } from 'expo';
+import { Platform, Pressable, StyleProp, View, ViewStyle } from 'react-native';
+
+export type ListProps = {
+  /**
+   * Custom styles for the progress component.
+   */
+  style?: StyleProp<ViewStyle>;
+
+  children?: React.ReactNode;
```

Comment:
could we type it in a way that it only accepts ListItem components? (not sure if possible)